packages/api-explorer/src/components/SideNav/SideNavMethods.tsx

```diff
@@ -85,12 +85,15 @@ export const SideNavMethods = styled(
           {Object.values(methods).map((method) => (
             <li key={method.name}>
               <Link
-                to={`${buildMethodPath(
-                  specKey,
-                  tag,
-                  method.name,
-                  searchParams.toString()
-                )}`}
+                to={() => {
+                  searchParams.delete('v')
```

Comment:
Relates to prior comment about removing parameters from the URL in a cleaner fashion, pointing out for reviewers